recipes/ux.rb

```diff
@@ -74,7 +74,7 @@
 end
 
 log 'Set ENV settings'
-template '/opt/ux-master/public/appConfig.js' do
+template '/home/jellyfish/ux/public/appConfig.js' do
```

Comment:
The path should not be hard coded.  It should be using ['rbenv']['install_prefix']